src/main/java/org/jenkinsci/plugins/workflow/cps/UninstantiatedDescribableWithInterpolation.java

```diff
@@ -0,0 +1,30 @@
+package org.jenkinsci.plugins.workflow.cps;
+
+import org.jenkinsci.plugins.structs.describable.UninstantiatedDescribable;
+
+import java.util.Map;
+import java.util.Set;
+
+public class UninstantiatedDescribableWithInterpolation extends UninstantiatedDescribable {
```

Comment:
We should restrict this class (some imports will need to be added along with this suggestion):

```suggestion
@Restricted(NoExternalUse.class)
public class UninstantiatedDescribableWithInterpolation extends UninstantiatedDescribable {
```